src/ca65/main.c

```diff
@@ -307,6 +307,14 @@ static void SetSys (const char* Sys)
             NewSymbol ("__OSIC1P__", 1);
             break;
 
+        case TGT_CREATIX_EMUL:
+            NewSymbol ("__CREATIX_EMUL__", 1);
+            break;
+            
+        case TGT_CREATIX_NATIVE:
```

Comment:
On 05/09/15 20:38, greg-king5 wrote:
>
> The emulation layer is needed by programs that are written in C (the 
> compiler doesn't generate the C39 addressing modes). Assembly-only 
> programs might not need that layer; therefore, it wouldn't need to be 
> installed in the PROM.
>


Yes, but do assembly programs need the C runtime lib?